compiler/src/main/scala/org/scalajs/core/compiler/GenJSCode.scala

```diff
@@ -5451,13 +5428,18 @@ abstract class GenJSCode extends plugins.PluginComponent
   private def isStringType(tpe: Type): Boolean =
     tpe.typeSymbol == StringClass
 
-  protected lazy val isHijackedBoxedClass: Set[Symbol] = {
-    /* This list is a duplicate of ir.Definitions.HijackedBoxedClasses, but
+  protected lazy val isHijackedClass: Set[Symbol] = {
+    /* This list is a duplicate of ir.Definitions.HijackedClasses, but
```

Comment:
Do we still need `ir.Definitions.HijackedBoxedClasses`? If yes, why?

Reply:
Right now, there are still two cases in the Emitter that use `ir.Definitions.HijackedBoxedClasses`. However that's limited to how the emitter does its job, and that definition should definitely be moved over there. However, I suggest we don't do that in this PR as it is not directly related. There are other things in `Definitions` that should be removed, btw.

I've filed #3195.